packages/core/vite.config.ts

```diff
@@ -9,7 +9,7 @@ export default defineConfig({
     },
   },
   test: {
-    include: ['src/**/*.spec.ts'],
+    include: ['src/__tests__/*.spec.ts'],
```

Comment:
This misses nested tests, which is probably going to confuse someone in the future. I think this change should be reverted or modified to add an extra `**`:

```suggestion
    include: ['src/**/__tests__/*.spec.ts'],
```